share/doc/wake/tutorial.md

```diff
@@ -586,8 +586,8 @@ in case the job failed. There is other information we can get from a `Job`
 object: `getJobStatus` is an `Integer` equal to the job's exit status,
 `getJobOutputs` returns a `List` of `Paths` created by the job, among others.
 
-`version` is just a `String` with the current wake version, and `here` a
-`String` with the location of the wake file.  We also have an example of a
+`version` is just a `String` with the current wake version, and `@here` is a
+`String` with the directory of the Wake file.  We also have an example of a
```

Comment:
not a huge deal, but my personal preference is `wake` over `Wake`

Reply:
Ah. Gotcha. I was operating under the personal rubric of Wake as a proper noun (and thus capitalizing it) when referring to it in general but then wake or `wake` when referring to the specific command line tool and maybe even `.wake` when referring to a file in some cases. (Not saying that's correct.) So I made the assumption it was a typo switching between those use cases. 🙃

However, I'm happy adjusting it back to stick with convention and then sticking with the agreed upon convention in other edits!

Reply:
[Fixed here to maintain current consistency in the tutorial.](https://github.com/sifive/wake/pull/1009/commits/58893f2f8ab42c2b1c0a2a21efec0d45347d1436) I noticed I wasn't even consistent in the same sentence with my update. Haha. Sorry about that.

The quick reference seems to use a mix, so some sections have Wake and others have wake, so I left my additions as Wake as the two previous sections (written by someone else) use Wake.